toolchain/lex/tokenized_buffer.h

```diff
@@ -64,15 +64,18 @@ using TokenIterator = IndexIterator<TokenIndex>;
 // buffer locations.
 class TokenDiagnosticConverter : public DiagnosticConverter<TokenIndex> {
  public:
-  explicit TokenDiagnosticConverter(const TokenizedBuffer* buffer)
-      : buffer_(buffer) {}
+  explicit TokenDiagnosticConverter(const TokenizedBuffer* tokens)
+      : tokens_(tokens) {}
 
   // Map the given token into a diagnostic location.
```

Comment:
This could also describe the int32_t in the output?

Reply:
Changing comment to note it's just implementing a virtual function, to avoid duplicating/diverging documentation.